dodo.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
```

Comment:
What's this file for, exactly?

Reply:
That's  to `doit` what `Makefile` is to `make`: your entry point